hs_composite_resource/views.py

```diff
@@ -0,0 +1,53 @@
+from django.http import JsonResponse
+from django.contrib.auth.decorators import login_required
+
+from rest_framework import status
+
+from hs_core.views.utils import ACTION_TO_AUTHORIZE, authorize, get_coverage_data_dict
+
+
+@login_required
+def update_resource_coverage(request, resource_id, coverage_type, **kwargs):
```

Comment:
Is coverage_type a necessary parameter?  Can't we just call `update_coverage`?

Reply:
Can we have both coverage types?  I thought it was one or the other for a resource.

Reply:
For composite resource there are 2 different buttons on the UI to independently update temporal and spatial coverage.